lib/ansible/config/ansible_builtin_runtime.yml

```
@@ -7759,7 +7759,7 @@ plugin_routing:
     network.common.utils:
       redirect: ansible.netcommon.network.common.utils
     network.dellos10.dellos10:
-      redirect: dellemc_networking.os10.network.os10
+      redirect: dellemc.os10.network.os10
     # FIXME: no collection source found for dellos6/dellos9
```

Here you need to add more entries:
```suggestion
    network.dellos6.dellos6:
      redirect: dellemc.os6.network.os6
    network.dellos9.dellos9:
      redirect: dellemc.os9.network.os9
```

---

@felixfontein Updated